openedx/features/course_experience/templates/course_experience/course-messages-fragment-proversity.html

```diff
@@ -0,0 +1,41 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import get_language_bidi
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.js_utils import js_escaped_string
+from openedx.core.djangolib.markup import HTML
+from openedx.features.course_experience import CourseHomeMessages
+%>
+
+<%
+is_rtl = get_language_bidi()
+%>
+
+% if course_home_messages:
+% for message in course_home_messages:
```

Comment:
could you indent this block  in order to improve the readability

Reply:
Addressed.